### DISCUSSION TITLE
Is connectivity checked or "host is reachable" as part of the networking

### DISCUSSION THREAD

**Discussion by stephenhauck** [nylo-core/nylo#111] [Q&A] [answered]
I always need to check to see not only are we connected but is the host reachable before trying to make calls over the network .. Is this built in to the networking feature of the framework ?

**Comment by agordn52** [comment, marked as answer]
Hi @stephenhauck,

Thanks for reaching out. Currently, Nylo will attempt the API request without checking the device's connectivity status.
You could use a package like [connectivity_plus](https://pub.dev/packages/connectivity_plus) or [connectivity_wrapper](https://pub.dev/packages/connectivity_wrapper) to help you achieve this in your project.

I do think this would be a nice addition to Nylo though, so I'll do a bit of thinking on how this could be integrated.

**Reply by stephenhauck** [reply]
Thanks for confirming that it would need to be done and is not currently part of the framework!
